### PR TITLE
Mailboxes use wooden posts

### PR DESCRIPTION
**Pull request by bloodbowel**
#### Summary
Features "Mailboxes now use wooden posts"

#### Purpose of change
I was looking at the description for the mailbox mentioning a wooden post in its construction. The wooden post description says it's often used for mailboxes too. But this furniture exists from before posts were added and still uses planks.

#### Describe the solution
I made the construction recipe and terrain item use a wooden post instead of a plank to stand on. I also made it drop more splintered wood when smashed.

#### Describe alternatives you've considered
None. This seems pretty uncontroversial to me.

#### Testing
Tested it on my computer in debug mode.

#### Additional context
Screenshot1
Screenshot2

### CODE REVIEW

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 1]
```diff
@@ -492,7 +492,12 @@
     "required_str": -1,
     "flags": [ "TRANSPARENT", "FLAMMABLE_HARD", "CONTAINER", "PLACE_ITEM", "MOUNTABLE" ],
     "deconstruct": {
-      "items": [ { "item": "2x4", "count": 2 }, { "item": "nail", "charges": [ 2, 5 ] }, { "item": "sheet_metal", "count": 1 } ]
+      "items": [
+        { "item": "2x4", "count": 1 },
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/furniture-storage.json:496]:
```suggestion
```
It's no longer a 2x4 when you cut it in half to create the base that goes on top of the wooden post to hold the mailbox.  I know you didn't create the original recipe.  It probably needs a 2x4s worth of wooden scraps which I guess are just splinters?

File: data/json/furniture_and_terrain/furniture-storage.json [hunk 2]
```diff
@@ -503,7 +508,7 @@
       "items": [
         { "item": "2x4", "count": 1 },
```
Comment (Maleclypse) [inline data/json/furniture_and_terrain/furniture-storage.json:509]:
```suggestion
```
Same as above

**Comment by Maleclypse** [comment]
Good catch!

**Review by bloodbowel — COMMENTED** [review]
That seems right. Thanks.

**Pull request merged**